schema/verify/computed_columns/form_result_has_unresolved_comments.sql

```diff
@@ -2,6 +2,16 @@
 
 begin;
 
-select pg_get_functiondef('ggircs_portal.form_result_has_unresolved_comments(ggircs_portal.form_result)'::regprocedure);
+do $$
+  begin
+
+    if (select exists(select * from pg_proc where proname='ggircs_portal.form_result_has_unresolved_comments')) then
+      raise exception 'ggircs_portal.form_result_has_unresolved_comments exists when it should not';
+    else
+      perform true;
+    end if;
+
+  end;
+$$;
```

Comment:
we can use the `ggircs_portal_private.verify_function_not_present( ... )` helper for all these sqitch verify procedures

Reply:
Thanks Pierre, I wrote this before your verify_function_not_present was merged. I'll refactor.

Reply:
ahh, I can't refactor. My files exist before that function is created

Reply:
Hard to keep track of which branch started when, it's definitely not an essential improvement

Reply:
Reordering the changes in the sqitch plan is an option, as long as it's all after the latest tag